source/develop/release-management/features/storage/backup-storage-domain.html.md

```diff
@@ -0,0 +1,248 @@
+
+
+## GSOC
+
+This feature is being developed as part of the Google Summer of Code (GSOC).
+Google Summer of Code is a global program focused on introducing students to open source software
+development. Students work on a 3 month programming project with an open source organization
+during their break from university.<br />
+The oVirt organization has chosen Shubham Dubey, a student from The LNM Institute of Information Technology from India, to work  on this project for the upcoming summer and Maor Lipchuk, a senior software developer in the oVirt storage team, to be the mentor.
+
+
+## Backup Storage Domain - Functionality
+
+* You can select any data storage domain as backup storage.
+* Once the storage domain is configured as backup the engine will block any running VMs or any changes that might be in the storage domain.
+* A backup storage domain can be detached and attached to a data center as every data storage domain.
+* Backup storage domain will be able to support unregistered VMs/Templates/Disks.
+* The backup indication will only be configured using the DataBase and will not be configured in the storage domain metadata - The user will be able to configure a storage domain as backup once a data storage domain is being imported or added to oVirt.
+* There is no obligation regarding data center version in oVirt to a backup storage domain. A backup storage domain can be part of every Data Center version.
+* A user can run a VM based on a Template which one of its disks is part of a backup storage domain while all the VM's disks are not part of the backup storage domain.
+
+#### Advantages
+
+* Much more dedicated storage domain for backup and disaster recovery purposes.
+* One step process-If you want a backup of VM/Template then just transfer it to backup storage domain.
+* For huge amount of VMs/Templates and OVFs migration the time taken for copying through export will be high which will be minimized by using backup storage domain.
+* Large amount of extra space is used in export storage domain. It may be suitable for small datacenter but for large datacenter the storage required to store those OVFs and VMs in export storage domain will become a headache.
+* Multiple storage domain can be used for backup purposes only.
+* Backup storage domain will support both File storage(NFS, Gluster) as well as block storage(Fiber Channel and iSCSI) as compare to export storage domain which only has support for file storage.
+
+#### Restrictions
+
+* A data storage domain can not be configured as backup while there are running VMs with disks reside on that storage domain.
+* User can not run VMs with disks reside on a storage domain configured as backup, since running VM might manipulate the disk's data.
+* VMs with disks reside on a backup storage domain can not be previewed.
+* Live move of disks to the backup storage domain will be restricted.
```

Comment:
Additionally: 

1. A backup domain cannot be elected as the master domain
2. The Hosted Engine's domain can't be configured as a backup domain (although this may be redundant as it always has the hosted engine VM which is up, but better be safe than sorry
3. The backup domain cannot be the target of memory volumes
4. The backup domain cannot hold VM leases (need to think about this - perhaps we should allow creating them but not running VMs with them?)

Open question - can you edit a VM/disk that's on the backup domain?